migrations/archive/2024/2024_purge_invite_accepted.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-console */
+const MIGRATION_NAME = '2024_purge_invite_accepted';
+import { model as User } from '../../../website/server/models/user';
+
+const progressCount = 1000;
+let count = 0;
+
+async function updateUser (user) {
+  count++;
+
+  return await User.updateOne(
```

Comment:
Would it make sense to use updateMany here? Could update all queried users at once in one call. (would lower query size maybe) should still make this a lot faster to run which would allow us to move the cutoff date back further to catch more users.

Reply:
Gave it a go!